### DISCUSSION TITLE
Potential error in SNE equivalence equation in the paper

### DISCUSSION THREAD

**Discussion by InfProbSciX** [lmcinnes/umap#813] [Q&A] [answered]
In the [arxiv paper](https://arxiv.org/pdf/1802.03426.pdf), page 57, equation 15, shouldn't the sign before rho_i be positive? Just checking if this is an error or a lapse in understanding on my end.

**Comment by jlmelville** [comment, marked as answer]
Yes you are correct. I am responsible for that mistake appearing in the paper. My apologies.

**Reply by InfProbSciX** [reply]
No problems, thanks!
